pure.zsh

```diff
@@ -176,6 +176,10 @@ prompt_pure_precmd() {
 	# preform async git dirty check and fetch
 	prompt_pure_async_tasks
 
+	# store name of virtualenv in psvar if activated
+	psvar[12]=
+	[[ -n $VIRTUAL_ENV ]] && psvar[12]="${VIRTUAL_ENV:t}"
```

Comment:
I had a thought: what if we check `[[ -n $VIRTUAL_ENV ]] && [[ $VIRTUAL_ENV_DISABLE_PROMPT = 1 ]]` here? My thought is that this would allow users to opt-out of us managing the virtualenv by e.g. `unset VIRTUAL_ENV_DISABLE_PROMPT`.